.gitignore

```diff
@@ -1 +1,2 @@
 t/servroot
+/.idea/
```

Comment:
Please don't include your local editor ignores in the repo. Use `git config --global core.excludesFile`.